Solutions/Shodan/Playbooks/CustomConnector/ShodanCustomConnector/readme.md

```diff
@@ -0,0 +1,45 @@
+# Shodan Logic App Custom Connector
+
+This custom connector connects to Shodan endpoint to execute actions supported by Shodan and returns back response in JSON format.
+
+### Authentication methods supported by this connector
+
+* API Key Authentication
+
+### Prerequisites to deploy Custom Connector 
+- Have Shodan API Endpoint Url handy (Default: https://api.shodan.io)
+
+### Deployment Instructions
+
+1. To deploy Custom Connector, click the Deploy to Azure button. This will launch the ARM Template deployment wizard.
+2. Fill in the required parameters:
+    - Connector Name: Enter the custom connector name (Default: ShodanCustomConnector)
+    - Service Endpoint: Enter the Shodan API endpoint (Default: https://api.shodan.io). Make sure to prefix with https://.  
```

Comment:
In custom connector https:// and http:// is getting replaced by " "   then why asking customer to prefix with https://

"host": "[replace(replace(parameters('ServiceEndpoint'),'https://',''),'http://','')]",